storage/innobase/xtrabackup/test/t/bug1192834-rollback.sh

```diff
@@ -0,0 +1,44 @@
+##########################################################################
+# Bug 1192834: Crash during apply with index compaction enabled          #
+##########################################################################
+
+. inc/common.sh
+
+start_server --innodb_file_per_table
+load_dbase_schema sakila
+load_dbase_data sakila
+
+function start_uncomitted_transaction()
+{
+    run_cmd $MYSQL $MYSQL_ARGS sakila <<EOF
+START TRANSACTION;
+DELETE FROM payment;
+SELECT SLEEP(10000);
+EOF
+}
+
+start_uncomitted_transaction &
+job_master=$!
+
+sleep 2
+
+backup_dir="$topdir/backup"
+
+innobackupex --no-timestamp --compact $backup_dir
```

Comment:
Do you use `innnobackupex` here intentionally? If yes, please put a comment why, IIRC it is going to be deprecated.